ILSANG/Sources/Views/Ranking/RankingItemView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  RankingItemView.swift
+//  ILSANG
+//
+//  Created by Kim Andrew on 10/7/24.
+//
+
+import SwiftUI
+
+struct RankingItemView: View {
+    
+    var level: Int
+    var nickName: String
+    var stat: String
+    var point: Int
```

Comment:
```suggestion
    let level: Int
    var rank: Rank
```
이런 식으로 객체로 가지고 있으면 초기화부분이 깔끔할 것 같습니다 ‼️
그리고 내부에서 바뀔 일이 없으니 최대한 let으로 선언해보면 좋을 것 같습니다 !

Reply:
수정 완료